omero/sysadmins/index.rst

```diff
@@ -2,6 +2,10 @@
 System Administrator Documentation
 ##################################
 
+This documentation begins with information aimed at OS-level administrators
+and moves on to day-to-day management of OMERO for facility
+administrators/managers (who may find it useful to read the :help:`Facility Managers help guide <facility-manager.html>` for an overview first).
```

Comment:
Never heard the word ``facility administrator`` before - it sounds like a forced transformation from ``system administrator`` (not sure though how to replace now)

Reply:
I guess I was thinking facility-level admin as opposed to operating system admin - I could add that clarification or remove the term completely if you prefer?

Reply:
I think so, thank you